descriptors/ModuleDescriptor-template.json

```diff
@@ -410,6 +410,17 @@
         }
       ]
     },
+    {
+      "id": "orders-storage.receiving_history",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/orders-storage/receiving_history",
+          "permissionsRequired": ["orders-storage.receiving_history.collection.get"]
```

Comment:
As discussed in the Slack, we are going to start to follow [FOLIO Naming conventions](https://dev.folio.org/guidelines/naming-conventions/#permissions) as much as we can. So probably it makes sense to start applying them at least for the new changes:
> The naming scheme is a faceted dot-separated string, with the delimited terms as dash-separated words.